Autopilot/Path Manager/PathManager.c

```diff
@@ -667,6 +674,19 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+static void checkForFirstGPSLock(){
+	static char gpsLockFlag = 1;
```

Comment:
In this case it wont, since you're resetting it at the begining of the function. It would retain its value, if say, you only declared it at the top (not set it), and then set it through some form of if conditions, which wouldnt work in this case.